main.py

```diff
@@ -23,27 +23,20 @@
 DOCTOLIB_BASE_API_URL = "https://www.doctolib.de/availabilities.json"
 
 PRACTICE_PARAMS = {
-    "biontech_arena": "visit_motive_ids=2495719&agenda_ids=397766-397800-402408-397776&practice_ids=158431",
-    "biontech_messe": "visit_motive_ids=2495719&agenda_ids=397844-397846-457504-397845-457511-457405-457406-457411-457415-457416-457418-457426-457436-457439-457443-457453-457477-457487-457497-404659-457400-457404-457407-457408-457409-457410-457412-457414-457419-457420-457421-457424-457425-457427-457428-457429-457430-457432-457435-457442-457448-457457-457463-457470-457483-457489-457493&practice_ids=158434",
```

Comment:
Well yes. I just updated the script for my own needs and kicked out the out-of-date requests for first and second vaccination.

In that sense: this PR is not necessarily a great contribution. I was just looking up the new api ids and wanted to share them in case anyone else is interested. I have my script up and running now so no hard feelings if you don't want to merge this :D I hope to never need this again in a few weeks anyway ^^

Reply:
@hexaeder do not get me wrong here, I am actually happy for this contribution and that this helps you to get what you need. Just wonder if we can make it even better and if someone still might need the original version to grab first/second shots...
However, knowing our vaccination rates, I am now thinking that we don't really need to get first shot appointments anymore. :D